src/pages/home/Home.js

```diff
@@ -1,9 +1,80 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './Home.css';
+import axios from 'axios';
+import logo from '../../assets/logo.png';
+import {Link} from "react-router-dom";
+import Header from "../../components/header/Header";
+import textSlicer from "../../helpers/textSlicer";
+
+function Home() {
+    const [posts, setPosts] = useState([]);
+    const [error, setError] = useState(false);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => {
+        const controller = new AbortController();
+
+        async function fetchData() {
+            setLoading(true);
+
+            try {
+                setError(false);
+                const results = await axios.get('https://www.reddit.com/hot.json?limit=15',{signal: controller.signal});
+                console.log(results);
+                const fifteenPosts = results.data.data.children;
+                setPosts(fifteenPosts);
+                console.log(fifteenPosts);
+
+            } catch (e) {
+                console.error(e);
+                setError(true);
+            }
+            setLoading(false);
+        }
+        fetchData();
+
+        // return function cleanup() {
+        //     controller.abort();
+        // }
+    }, []);
 
-function Home(props) {
     return (
-        <div></div>
+        <>
+            {error && <p>Error: could not fetch data!</p>}
+            {loading && <p>Loading...</p>}
+
+            <Header>
+                <img src={logo} alt='logo Reddit'/>
+                <h1 className='business-name'>Reddit</h1>
+            </Header>
+            <main className='outer-container'>
+                <div className='inner-container headings'>
+                    <h2>Hottest posts</h2>
+                    <h4>on Reddit right now</h4>
+                    <div className='flex-wrap'>
+                        {posts &&
+                            <>
+                                {posts.map((post) => {
```

Comment:
1. De eerste check is in principe overbodig, omdat die altijd true zal zijn
2. De lege tags zijn niet nodig, omdat het maar één child bevat

```suggestion
                        {posts.map((post) => {
```